int8 car_0
int8 car_1
bool ready_0
bool ready_1
